inclusive-design/idrc
# feat: set up event promo section

## Description

Add promo for ACAW/CAMA conference.

## Steps to test

1. Visit home page.

**Expected behavior:** Banner [appears as designed](https://www.figma.com/file/GyPtCZtqRWxpoKLPEMn05d/IDRCWebsite_Screens_2022---Projects-pages-revision?type=design&node-id=2614-5749&mode=design&t=0ix5UDzOTvF19Rjv-4).

## Additional information

Not applicable.

## Related issues

Not applicable.
